bc/channel/models.py

```diff
@@ -1,13 +1,36 @@
 from django.db import models
 
 from bc.core.models import AbstractDateTimeModel
+from bc.sponsorship.models import Sponsorship
 from bc.users.models import User
 
 from .utils.connectors.base import BaseAPIConnector
 from .utils.connectors.masto import MastodonConnector, masto_regex
 from .utils.connectors.twitter import TwitterConnector
 
 
+class Alias(AbstractDateTimeModel):
+    name = models.CharField(
+        help_text="Name for a set of channels",
+        max_length=100,
+    )
+    is_big_cases = models.BooleanField(
+        help_text="Designates whether this alias should be treated as the group of big cases channels",
+        default=False,
+    )
+    sponsorships = models.ManyToManyField(
+        Sponsorship,
+        related_name="channel_groups",
```

Comment:
When a related name doesn't match the name of the class, that's usually a smell. It usually means that you didn't like the name of the model and are thinking of it a different way.

I think in this case, calling these objects aliases doesn't really make sense. An alias a different name for something. These really are more like channel groups, right?